packages/layouts-css/src/breakpoints.scss

```diff
@@ -1,31 +1,31 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @mixin mobile {
-  @media (max-width: 486px) {
+  @media (max-width: 31.25rem) { //500px
     @content;
   }
 }
 
 @mixin landscape-mobile {
-  @media (min-width: 487px) and (max-width: 768px) {
+  @media (max-width: 68.75rem) { //1100px
     @content;
   }
 }
 
 @mixin tablet {
-  @media (min-width: 769px) and (max-width: 991px) {
+  @media (max-width: 68.75rem) { //1100px
     @content;
   }
 }
 
 @mixin small-monitor {
-  @media (min-width: 992px) and (max-width: 1399px) {
+  @media (max-width: 93.75rem) { //1500px
     @content;
   }
 }
 
 @mixin monitor {
-  @media (min-width: 1400px) {
+  @media (min-width: 93.75rem) { //1500px
```

Comment:
Breakpoints should only travel in one direction, so use either `max-width` for all breakpoints or `min-width` for all of them.

this means `monitor` could be removed

Reply:
But there are some places where we have additional stylings for monitor, i.e in `grid.scss` :
 ` @include monitor {
    grid-column: var(--_iui-grid-item-column-start-monitor) /
      var(--_iui-grid-item-column-span-monitor);
  }`

So should we just remove the `min-width` but still keep the mixin?

Reply:
those styles can go outside the other breakpoints. they will be applied when none of the other breakpoints match

```scss
.example-class {
  // >1500px styles

  @include mobile {
    // <500px styles
  }

  @include tablet {
    // 500px-100px styles
  }

  @include small-monitor {
    // >1100px styles
  }
}
```

Reply:
Thanks, I updated this